app/forms/import_form.rb

```diff
@@ -0,0 +1,32 @@
+class ImportForm
+  include ActiveModel::Model
+  attr_accessor :url, :preview, :rdf_importer
+  delegate :term_list, :to => :rdf_importer
+
+  def initialize(params = {})
+    super(params)
+    @rdf_importer = RdfImporter.new(errors)
```

Comment:
Easy enough, but I have the same concern here as above - should the controller (or its injector) be in charge of how its dependency does its job?  It feels like we're just coupling things in a less direct way if we go that route.